style.css

```diff
@@ -0,0 +1,296 @@
+}
+
+.headline-section p {
+  color: #344563;
+  width: 60%;
+  margin: 10px;
+}
+
+.headline-section ul {
+  padding: 10px;
+  color: #505f79;
+  width: 60%;
+  display: flex;
+  list-style-type: none;
+}
+
+.headline-section ul li {
+  padding: 5px;
+  margin-right: 10px;
+}
+
+.text {
+  text-decoration: none;
+  font-size: 16px;
+  margin-left: 10px;
+  color: #7f8cff;
+}
+
+.btn {
+  border: none;
+  background-color: #fff;
+  color: #6070ff;
+  font-size: 30px;
+  margin-left: 70px;
+}
+
+.work-section {
+  color: #091e42;
+  max-width: 500px;
+  margin: 0 auto;
+  padding: 40px 30px;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 50px;
+  text-align: right left;
+}
+
+.workd-card h1 {
+  color: #172b4d;
+  font-weight: bold;
+  font-size: 2rem;
+  width: 60%;
+  margin: 10px;
+  margin-top: 10px;
+  text-align: right left;
+}
+
+.work-section p {
+  color: #344563;
+  width: 70%;
+  margin: 10px;
+}
+
+.big-button {
+  padding: 10px;
+  border-radius: 8px;
+  color: #396df2;
+  text-align: center;
+  background-color: #fff;
+  border: 1px solid #6070ff;
+  font-weight: bolder;
+  width: 30%;
+  margin-left: 15px;
+  margin-bottom: -20px;
+  font-size: 17px;
+  letter-spacing: 3%;
+}
+
+.list-container-1 {
+  color: #505f79;
+  display: flex;
+  justify-content: space-around;
+  width: 60%;
+  margin: 10px;
+  list-style-type: none;
+}
+
+.list-container-1 img {
+  border-radius: 50%;
+  width: 8px;
+  height: 8px;
+  margin: 3px;
+}
+
+.list-container-2 ul {
+  display: flex;
+  list-style-type: none;
+  color: #6070ff;
+  margin-bottom: 10px;
+}
+
+.list-container-2 ul li {
+  background-color: #ebebff;
+  border-radius: 12%;
+  padding: 7px;
+  text-align: center;
+  margin: 10px;
+}
+
+.canopy {
+  color: #344563;
+  text-transform: uppercase;
+  font-weight: bold;
+}
+
+.about-myself h1 {
+  width: 30%;
+  margin: 10px;
+}
+
+.about-myself p {
+  width: 327px;
+  margin: 10px;
+}
+
+.about-myself ul {
+  padding: 10px;
+  color: #505f79;
+  width: 60%;
+  display: flex;
+  list-style-type: none;
+}
+
+.about-myself ul li a {
+  color: #505f79;
+  text-decoration: none;
+  font-size: 19px;
+}
+
+.about-myself ul li {
+  padding: 5px;
+  margin-right: 10px;
+}
+
+.down-arrow {
+  display: flex;
+  justify-content: space-between;
+}
+
+.down-arrow i {
+  color: #505f79;
+  margin-left: 100px;
+}
+
+.space {
+  margin-bottom: 30px;
+}
+
+#text-color {
+  color: #396df2;
+  margin: 10px;
+}
+
+.below-button {
+  width: 40%;
+}
+
+.below-button:hover {
+  background-color: #6070ff;
+  color: white;
+}
+
+.frames {
+  color: #172b4d;
+  width: 60%;
+  margin: 10px;
+  margin-top: 10px;
+  text-align: right left;
+}
+
+.alignment {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+/* Conatact */
+.contact {
+  background-image: url(images/Contact\ me.png);
+  background-repeat: no-repeat;
+  flex-wrap: wrap;
+  max-width: 400px;
+  margin: 0 auto;
+  padding: 40px 30px;
+  text-align: right left;
+}
```

Comment:
- [ ] Please 🙏🏽 make sure that you are using Flexbox `display: flex `to lay out the elements of the page. It is a requirement:

> In order to lay out the elements on the page you should use Flexbox.